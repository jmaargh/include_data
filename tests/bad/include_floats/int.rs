use include_data::include_f32;

fn main() {}

// Can't use this macro for ints, even of the same size
static INTS: &[u32] = include_f32!("../../test_data/binary_64");
