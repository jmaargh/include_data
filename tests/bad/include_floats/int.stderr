error[E0308]: mismatched types
 --> tests/bad/include_floats/int.rs:6:23
  |
6 | static INTS: &[u32] = include_f32!("../../test_data/binary_64");
  |                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected `u32`, found `f32`
  |
  = note: expected reference `&'static [u32]`
             found reference `&'static [f32]`
  = note: this error originates in the macro `$crate::include_floats` which comes from the expansion of the macro `include_f32` (in Nightly builds, run with -Z macro-backtrace for more info)
